skfda/misc/metrics/_sdtw_numba.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from numba import njit
+
+
+@njit(cache=True, fastmath=True)
```

Comment:
`fastmath` breaks IEE 754 compliance. Is that safe to use here?

Reply:
At least in ``_sdtw`` function i think it can be set to False since, except in ``_soft_min_argmin``, their are only linear operations. Thus, intuitively, it is safer to set to True for ``_soft_min_argmin`` as there log and exp functions.